tutornotes/templates/notes/build/notes/Dockerfile

```diff
@@ -1,24 +1,23 @@
 # syntax=docker/dockerfile:1.4
-FROM docker.io/python:3.12-slim-bookworm
+FROM docker.io/ubuntu:24.04
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
   --mount=type=cache,target=/var/lib/apt,sharing=locked \
   apt update && \
   apt upgrade -y && \
   apt install -y \
-    locales \
+    language-pack-en \
     git \
-    python3-dev \
-    build-essential \
-    default-libmysqlclient-dev \
-    pkg-config && \
-  sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && \
-  locale-gen
-ENV LC_ALL=en_US.UTF-8
+    python3 \
+    python3-pip \
+    python3-venv \
+    libmysqlclient-dev \
+    pkg-config
+RUN ln -s /usr/bin/python3 /usr/bin/python
 
 ###### Git-clone Notes repo ######
-ARG APP_USER_ID=1000
+ARG APP_USER_ID=1001
```

Comment:
Please take a look at https://github.com/overhangio/tutor/pull/1117#discussion_r1795117242 for context on this change.

Reply:
@DawoudSheraz addressed